cmd/notification.go

```diff
@@ -185,12 +182,12 @@ func (sys *NotificationSys) LoadCredentials() []NotificationPeerErr {
 // StartProfiling - start profiling on remote peers, by initiating a remote RPC.
 func (sys *NotificationSys) StartProfiling(profiler string) []NotificationPeerErr {
 	var idx = 0
-	ng := WithNPeers(len(sys.peerRPCClientMap))
-	for addr, client := range sys.peerRPCClientMap {
-		client := client
+	ng := WithNPeers(len(sys.peerHosts))
+	for _, addr := range sys.peerHosts {
```

Comment:
We don't need idx anymore it was used with map previously, since now we have peerHosts as slice we can simply do 

```
for idx, addr := range sys.peerHosts {
```